computer/ui/styles.css

```
@@ -1,204 +1,460 @@
+  font-size: 12.5px;
+  color: var(--color-ink);
+  border-bottom: 1px solid var(--color-edge);
+}
+[data-iii-ui="computer"] .cp-ui-banner.warn {
+  background: var(--color-warn-muted);
+}
+[data-iii-ui="computer"] .cp-ui-banner.alert {
+  background: var(--color-alert-muted);
+}
+[data-iii-ui="computer"] .cp-ui-banner > span {
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)